tests/unit/src/test/scala/tests/ReportsSuite.scala

```diff
@@ -40,7 +39,7 @@ class ReportsSuite extends BaseSuite {
     val obtained =
       new String(Files.readAllBytes(path.get), StandardCharsets.UTF_8)
     assertNoDiff(exampleText(StdReportContext.WORKSPACE_STR), obtained)
-    assert(ReportFile.fromFile(path.get.toFile).nonEmpty)
+    assert(reportsProvider.incognito.getReports().length == 1)
   }
 
   test("delete-old-reports") {
```

Comment:
I wonder if it would be a good idea to have some tests for the log file management as well (size limit, backup, old files removal). :slightly_smiling_face:

Reply:
We could potentially generate a large enough file before beginning the test :thinking: might be good idea

Reply:
If it helps, the size limit could be injected into the `MetalsLogger` rather than hardcoded inside, so that the test would work with smaller files (faster, less resource intensive). :+1:

Reply:
We could have a server property for that, might be useful for contributors